org.civicrm.reqangular/test/services/api.spec.js

```diff
@@ -99,26 +103,84 @@ define([
         });
       });
 
+      describe('caching', function () {
+        var sampleResult = 'sample result';
+        var sampleParams = {};
+
+        beforeEach(function () {
+          spyOn($cacheFactory.get('$http'), 'remove').and.callThrough();
+        });
+
+        describe('when "returnCachedData" parameter is not passed', function () {
+          beforeEach(function () {
+            expectAndSendGET(sampleResult, sampleParams);
+            $rootScope.$digest();
+          });
+
+          it('does not flush cache before HTTP call', function () {
+            expect($cacheFactory.get('$http').remove).not.toHaveBeenCalled();
+          });
+
+          it('caches data by default', function () {
+            expect($cacheFactory.get('$http').info().size).toBe(1);
+          });
+        });
+
+        describe('when "returnCachedData" parameter passed as TRUE', function () {
```

Comment:
put `$cacheFactory.get('$http').info().size` in a well-named variable, so the expectation reads better and it's more expressive

Reply:
Done.